lib/actions/app.coffee

```diff
@@ -169,7 +191,15 @@ exports.associate =
 					return callback(new Error("Invalid application: #{params.name}"))
 
 				message = "Are you sure you want to associate #{currentDirectory} with #{params.name}?"
-				visuals.patterns.confirm(options.yes, message, callback)
+				if options.yes
+					return callback(null, true)
+				else
+					form.ask
+						message: message
+						name: 'confirm'
+						type: 'confirm'
+						default: false
+			    .nodeify(callback)
```

Comment:
@mkorakakis They can *seem* the same from a visual standpoint. I suggest you to enable "Show Invisibles" in Atom settings: https://gist.github.com/timrwood/9242517#gistcomment-1181001, so you can spot the differences in the future.